index.html

```diff
@@ -10,6 +10,7 @@
       rel="stylesheet"
       href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
     />
+    <script src="./script.js" defer></script>
```

Comment:
what is defer ?

Reply:
Defer let the javascript file load in background and execute the file when the HTML file is done parsing